functions/Clear-DbaSqlConnectionPool.ps1

```diff
@@ -44,31 +44,31 @@ function Clear-DbaSqlConnectionPool {
 		[Parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
 		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
-		[PSCredential]$Credential,
+		[PSCredential][System.Management.Automation.Credential()]$Credential = [System.Management.Automation.PSCredential]::Empty,
```

Comment:
We are actually reverting this one to just [PSCredential] as it caused issues. 

I'll update the spreadsheet on this one.